Cargo.toml

```diff
@@ -38,3 +38,7 @@ serde_json = "1.0"
 thiserror = "1.0"
 tiny-keccak = "1.5"
 web3 = "0.8"
+
+[dev-dependencies]
+anyhow = "1.0"
```

Comment:
How is this used?

Reply:
`dev-dependencies` are dependencies that only get used for testing, examples and benchmarking. I think doc tests also uses it.

Reply:
Right, I was wondering who uses anyhow, since I could not find it in the code.

Reply:
Oh! `async.rs` used to use it. I must have forgotten to remove the dependency